router/router.go

```diff
@@ -1021,9 +1021,9 @@ func (worker *workerT) updateReqMetrics(respStatusCode int, diagnosisStartTime *
 func (worker *workerT) allowRouterAbortedAlert(errorAt string) bool {
 	// when error occur during transformation(rt or batch) or when skipRtAbortAlertForTransformation is false(default)
 	allowTf := errorAt == routerutils.ERROR_AT_TF && !worker.rt.skipRtAbortAlertForTransformation
-	// (when error occur during delivery and (when proxy is not enabled or when skipRtAbortAlertForDelivery is false(default)))
+	// (when error occur during delivery and (when proxy is not enabled and when skipRtAbortAlertForDelivery is false(default)))
 	// or when destination is managed custom destination manager
-	allowDel := (errorAt == routerutils.ERROR_AT_DEL && (!worker.rt.transformerProxy || !worker.rt.skipRtAbortAlertForDelivery)) || errorAt == routerutils.ERROR_AT_CUST
+	allowDel := (errorAt == routerutils.ERROR_AT_DEL && (!worker.rt.transformerProxy && !worker.rt.skipRtAbortAlertForDelivery)) || errorAt == routerutils.ERROR_AT_CUST
```

Comment:
How about this?
```go
if errorAt == routerutils.ERROR_AT_CUST {
  return true
}
if worker.rt.skipRtAbortAlertForTransformation {
  return false
}
return errorAt == routerutils.ERROR_AT_TF ||
(errorAt == routerutils.ERROR_AT_DEL && !worker.rt.transformerProxy)
```

Reply:
Simplified it taking inspiration from your suggestion. Can you once take a look and let me know if this is more readable ?